apps/dashboard/app/models/motd_file.rb

```diff
@@ -37,12 +37,12 @@ def formatter
     case ENV['MOTD_FORMAT']
       when 'osc'
         @motd = MotdFormatterOsc.new(self)
-      when 'markdown'
-        @motd = MotdFormatterMarkdown.new(self)
+      when 'markdown_erb' || 'markdown'
+        @motd = MotdFormatterMarkdown.new(self, ENV['MOTD_FORMAT'])
```

Comment:
This is control coupling https://thoughtbot.com/blog/types-of-coupling. Create two new classes:

- MotdFormatterMarkdownErb
- MotdFormatterPlaintextErb

It might take more lines of code, but removes control flow and thus is easier to read.